scripting/executes/loadout_common.sp

```diff
@@ -42,7 +42,24 @@ public char RandomNade() {
   }
 }
 
-//
+public void GivePrimaryRifle(int client, int team) {
```

Comment:
Rename this SetPrimaryRifle - GivePrimaryRifle makes me thing it's actually giving the weapon (like GivePlayerItem). Other than that everything looks good to me.